PCbuild/python.vcxproj

```diff
@@ -132,9 +132,6 @@ set PYTHONPATH=$(PySourcePath)Lib
   </Target>
   <Target Name="GeneratePythonBat" AfterTargets="AfterBuild">
     <PropertyGroup>
```

Comment:
```suggestion
    <PropertyGroup>
      <_PGOPath Condition="$(Configuration) == 'PGInstrument'">@set PATH=%PATH%%3B$(OutDir)</_PGOPath>
```

Reply:
Leaving `$(_PGOPath)` like this fixes my errors. The way is more generic than  #32146.

Reply:
No, your other PR is better. We shouldn't rely on PATH for resolving DLLs, but should copy the right one alongside the executable.

Reply:
The additional change is perfect for PGO/non-PGO.
I can confirm PGInstrument builds work with this.